### ISSUE TITLE
Can I donate?

### ISSUE THREAD

**Issue by xeoncross** [airbornio/signed-web-apps#1]
You guys started something amazing here. Javascript crypto suffers from a threat model that puts trust in the server. This model fails if hackers, employees, providers, or governments or anyone else modify the server responses.

Where can I make a small donation?

**Comment by twiss** [comment]
Hey @Xeoncross, thanks so much for the kind words and for offering to donate!

I've whipped up a donation page here: https://www.airborn.io/donate

Let me know if it works for you!

**Comment by joshua-s** [comment]
@twiss it looks like you have a minimum $0.75 transaction fee. So, if someone donates below that amount, you will lose money. You might want to restrict this 😄

**Comment by xeoncross** [comment]
Thanks, though I am not willing to fill out all my personal information just to tip you. A simple paypal.me donate page or email is enough.

![screen shot 2018-04-02 at 2 49 50 pm](https://user-images.githubusercontent.com/56460/38213325-69346c4a-3686-11e8-9902-2f55ad983c6b.png)

**Comment by twiss** [comment]
@joshua-s Yeah. The payment processor enforces a minimum $1 transaction for that reason, and if you submit the page with a value lower than 1 your browser should complain (the input has `min="1"`). However, since that page uses your local currency, rather than always using $, there might be an edge case if you try to donate less than $1 worth of some other currency. The transaction will probably fail but I haven't tested it.

@Xeoncross Yeah, I understand. People have successfully paid using fake information in the past, so you could try that. Let me know if that doesn't work.

**Comment by xeoncross** [comment]
Um, instead of trying to fake information, can I just send money to `d.huigens@protonmail.com`?

**Comment by twiss** [comment]
I meant enter fake information into the form you posted above :) But if you prefer PayPal, mine is `d.huigens@gmail.com`. (Thanks in advance in any case!)
